plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -204,7 +204,7 @@ protected boolean filterSchema(String schemaName)
             try (ResultSet resultSet = getTables(connection, Optional.of(remoteSchema), Optional.of(remoteTable))) {
                 List<JdbcTableHandle> tableHandles = new ArrayList<>();
                 while (resultSet.next()) {
-                    tableHandles.add(new JdbcTableHandle(schemaTableName, getRemoteTable(resultSet)));
+                    tableHandles.add(new JdbcTableHandle(schemaTableName, getRemoteTable(resultSet), getTableComment(resultSet)));
```

Comment:
can we have `JdbcMetadata.getTableMetadata` invoke `getTableComment` without adding it to the handle?

Reply:
@findepi Reverted the change in handle and updated `JdbcMetadata` class.